### DISCUSSION TITLE
--legacy-peer-deps doesnt work

### DISCUSSION THREAD

**Discussion by MarkForLoop** [pnpm/pnpm#5487] [Q&A]
I am trying to  run
`pnpm i @react-pdf/renderer --legacy-peer-deps`

I  get this error
` ERROR   ERROR  Unknown option: 'legacy-peer-deps'
Did you mean 'strict-peer-dependencies'? Use "--config.unknown=value" to force an unknown option.`

![Screenshot 2022-10-13 at 17 19 39](https://user-images.githubusercontent.com/91071058/195651337-119fca44-ba4c-493e-9e5b-1d94fa7b3405.png)

so I ran 
`pnpm i @react-pdf/renderer strict-peer-dependencies`

![Screenshot 2022-10-13 at 17 21 22](https://user-images.githubusercontent.com/91071058/195651705-b72c91e2-7f42-4036-ad6a-2956d322441b.png)

am I doing this correctly???

**Comment by itskaransingh** [comment]
use pnpm add instead of pnpm i, I guess it should work

**Reply by yamanidev** [reply]
it doesn't ;(

**Comment by zkochan** [comment]
pnpm doesn't have such flag. pnpm automatically installs peer dependencies. If you don't want pnpm to automatically install peer dependencies, set [auto-install-peers](https://pnpm.io/npmrc#auto-install-peers) to `false`.

**Reply by yamanidev** [reply]
so essentially we just use the regular install command without that flag, and it would do what we expect from the flag, because it's the default behavior of pnpm?

**Reply by zkochan** [reply]
Yes, if the intended behaviour is to automatically install peers. npm CLI has changed the behaviour a few times, so it is hard to tell what you expect from "legacy peers".
